App/Models/SyncSessionModel.cs

```diff
@@ -0,0 +1,249 @@
+using System;
+using Coder.Desktop.App.Converters;
+using Coder.Desktop.MutagenSdk.Proto.Synchronization;
+using Coder.Desktop.MutagenSdk.Proto.Url;
+
+namespace Coder.Desktop.App.Models;
+
+// This is a much slimmer enum than the original enum from Mutagen and only
+// contains the overarching states that we care about from a code perspective.
+// We still store the original state in the model for rendering purposes.
+public enum SyncSessionStatusCategory
+{
+    Unknown,
+    Paused,
+    Error,
+    Conflicts,
```

Comment:
Worth a comment that categories higher than conflicts will get replaced by conflicts if there are any.